src/pages/Artist.jsx

```diff
@@ -77,22 +90,62 @@ function Artist() {
             {artistName}
           </h1>
         </div>
-
         <div className="h-full w-full">
-          <div className="flex h-full w-full items-center">
-            {artworks.map((artwork, i) => {
-              return (
-                <div key={artwork.id}>
-                  <img
-                    src={`https://www.artic.edu/iiif/2/${artwork.image_id}/full/400,/0/default.jpg`}
-                    alt={`${artwork.title}`}
-                    className={`${getClassName(
-                      i,
-                    )} absolute bottom-0 left-0 right-0 top-1/3 mx-auto h-[500px] w-[350px] object-cover opacity-0 grayscale duration-500`}
-                  />
-                </div>
-              );
-            })}
+          {artworks.map((artwork, i) => {
+            return (
+              <div key={artwork.id}>
+                <img
+                  src={`https://www.artic.edu/iiif/2/${artwork.image_id}/full/400,/0/default.jpg`}
+                  alt={`${artwork.title}`}
```

Comment:
```suggestion
                  alt={artwork.title}
```
La template string n'est pas obligatoire si y'a juste une string dedans